src/js/profile/mobile/widget/TimePicker.js

```diff
@@ -146,20 +159,31 @@
 				if (name === "hour") {
 					if (self.options.format === "24") {
 						options.min = 0;
-						options.max = 24;
+						options.max = 23;
 					} else {
-						options.min = 0;
+						options.min = 1;
 						options.max = 12;
 					}
 					spinContainer.classList.add(TimePicker.classes.HOUR_CONTAINER);
 				} else if (name === "minute") {
 					options.min = 0;
-					options.max = 60;
+					options.max = 59;
 					spinContainer.classList.add(TimePicker.classes.MINUTE_CONTAINER);
 				}
 
+				spinInput.min = options.min;
+				spinInput.max = options.max;
+				spinInput.type = "number";
+				spinInput.step = "1";
+				spinInput.classList.add(TimePicker.classes.TIME_INPUT);
```

Comment:
You can write just `classes.TIME_INPUT`

Reply:
I know it, but I used it intentionally. Because Spin classes are also used, I wanted to distinguish clearly. If it doesn't matter, I want to use it.